lldb/test/Shell/helper/toolchain.py

```diff
@@ -138,7 +191,7 @@ def use_support_substitutions(config):
     # Set up substitutions for support tools.  These tools can be overridden at the CMake
     # level (by specifying -DLLDB_LIT_TOOLS_DIR), installed, or as a last resort, we can use
     # the just-built version.
-    host_flags = ["--target=" + config.host_triple]
+    host_flags = ["--target=" + config.target_triple]
```

Comment:
Is it worth changing this variable name?

Naming is hard, but "host" sounds like the thing hosting the test runner, where now it refers to the thing that the tests run on. `target_flags` maybe?

Reply:
Hmm. I thought it was named after the substitution name %clang_host (used by tests not bound to the specific architecture).
I think it may be better to rename the substitution to %clang_target instead, but it requires a huge change of all tests.
Otherwise, we get the line like 
`config.substitutions.append(("%clang_host", "%clang " + target_flags))`
which also looks confusing IMO.